actions/lib/formatters.py

```diff
@@ -248,3 +252,16 @@ def decode_base64(data):
 def file_response_to_dict(response):
     result = {'commit': response['commit'].sha}
     return result
+
+
+def ref_to_dict(ref):
+    result = {
+        'object': {
+            'sha': ref.object.sha,
```

Comment:
The way (coding syntax) to create the result is different from the rest of this file.  It may be better to use the same coding style.

Reply:
I like the readability of this format, so I'm going to keep it for now 😁